src/app/components/client/ExposureCard.tsx

```diff
@@ -59,6 +60,7 @@ export type ExposureCardProps = {
   locale: string;
   fromEmail?: string;
   color: string;
+  featureFlagsEnabled: FeatureFlagsEnabled;
```

Comment:
I think I'd split this up into _just_ the feature flags we're interested in, or just pick those from that object, because you're constructing this object manually on the dashboard page. Otherwise, every time we add a new flag to `FeatureFlagsEnabled`, the dashboard page and all tests will have to be updated to add it as well, even if it's not at all relevant.

For example:

```suggestion
  premiumBrokerRemovalEnabled: boolean;
```

or

```suggestion
  featureFlagsEnabled: Pick<FeatureFlagsEnabled, "FreeBrokerScan" | "PremiumBrokerRemoval">;
```